WebHosts/WebHost.ClientApi/Characters/Models/CharacterName.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace WebHost.ClientApi.Characters.Models;
+
+public class CharacterName
+{
+    public string name { get; set; }
+    public string lang { get; set; }
```

Comment:
Can you use `PascalCase` for the properties? The API controller will automatically convert from `camelCase`.